### ISSUE TITLE
Emission part at HMMLearn.py

### ISSUE THREAD

**Issue by xuuuluuu** [DivyaGodayal/HMM-POS-Tagger#1]
When calculating emission matrix, you are writing below comments:
            # emission probability for a word-tag pair.
            # no. of occurrences of a given word-tag divided by no. of occurrences of the word.

Is it should be no. of occurrences of a given word-tag divided by no. of occurrences of the **tag**?

**Comment by DivyaGodayal** [comment]
Hey @xuuuluuu, thanks for pointing it out. It turns out that I had pushed an older version of the code when I was still experimenting with the model and somehow left this bug in the code. I have updated the code and corrected this. Here is the commit for the fix. 

[Commit](https://github.com/DivyaGodayal/HMM-POS-Tagger/commit/5a9cf03ab4c0abde2436f48721365ebbd307ac76
)
Hope this solves your doubt. 👍
